docs/use-cases/attachments.md

```diff
@@ -16,7 +16,7 @@ const msg = {
       filename: 'some-attachment.txt',
       type: 'plain/text',
       disposition: 'attachment',
-      content_id: 'mytext'
+      contentId: 'mytext'
```

Comment:
As per sendgrid open api specification, it is [content_id](https://github.com/twilio/sendgrid-oai/blob/main/spec/yaml/tsg_mail_v3.yaml#L601)

Reply:
The`contentId` property is used internally in the Attachment class. You are correct that the API expects the `content_id` property, and that is exactly what we send. The toJSON method of the Mail class handles this conversion, transforming the internal camelCase structure into the snake_case format expected by the API.

Reply:
@sbansla could you take a look, please?